smart_open/s3.py

```diff
@@ -477,8 +477,14 @@ def write(self, b):
 
         There's buffering happening under the covers, so this may not actually
         do any HTTP transfer right away."""
-        if not isinstance(b, six.binary_type):
-            raise TypeError("input must be a binary string, got: %r", b)
+        import sys
```

Comment:
Please move this stuff to module scope, and make it an internal constant, e.g. _BINARY_TYPES.

Reply:
Done